src/lib/secrets_helper.ts

```diff
@@ -133,20 +79,38 @@ const formatSecrets = (result: OneGraphSecretsResponse | undefined) => {
   return newSecrets
 }
 
-type OneGraphPayload = { authlifyToken: string | undefined }
-
-export type HandlerEventWithOneGraph = HandlerEvent & OneGraphPayload
+const logErrors = function (errors: GraphTokenResponseError[]) {
+  for (const error of errors) {
+    let errorMessage
```

Comment:
Are we only changing the function name for the error message?

Reply:
Yeah. I thought of making a function to create the error message, but thought it was overkill.